Firestore/Example/Tests/Core/FSTQueryListenerTests.m

```diff
@@ -170,9 +170,10 @@ - (void)testDoesNotRaiseEventsForMetadataChangesUnlessSpecified {
   FSTDocument *doc1 = FSTTestDoc(@"rooms/Eros", 1, @{@"name" : @"Eros"}, NO);
   FSTDocument *doc2 = FSTTestDoc(@"rooms/Hades", 2, @{@"name" : @"Hades"}, NO);
 
-  FSTListenOptions *options = [[FSTListenOptions alloc] initWithIncludeQueryMetadataChanges:YES
-                                                             includeDocumentMetadataChanges:NO
-                                                                      waitForSyncWhenOnline:NO];
+  FSTListenOptions *options =
```

Comment:
```
$ clang --version
Apple LLVM version 8.1.0 (clang-802.0.42)
Target: x86_64-apple-darwin16.7.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
```

I think I see what happened though. I (mistakenly) changed this file in 80a40bd and then backed out those changes in 58ccdba. However, the run of style.sh was in between.

If you like, I can revert this file completely. OTOH, style.sh thinks this is better, and since it's already fixed it... <shrug>

Reply:
Hmm.. thinking about this further, I'll just back this out.